src/main/java/org/helpberkeley/memberdata/RestaurantTemplateParser.java

```diff
@@ -106,7 +107,8 @@ private RestaurantBean nextRow() {
     protected abstract List<RestaurantBean> parse(String csvData);
     protected abstract boolean isAddressBlockMarker(RestaurantBean bean);
 
-    public Map<String, Restaurant> restaurants() {
+    public Map<String, Restaurant> restaurants(boolean validateFormulas) {
+        this.validateFormulas = validateFormulas;
```

Comment:
Boolean parameters are best avoided.  That make for code that you can't understand from the caller without going and looking at the implementation. true vs. false is not descriptive of what the parameter means.  So the preference, if it needs to be a parameter is to use an enum.  Note the difference of readability between:
`parser.restaurants(true);` and `parser.restaurants(VALIDATE_FORMULAS);`

However, in this case, adding the parameter doesn't look like the best choice.  We have 2 places in production and 3 in test cases that are passing true, and 16 (that are otherwise unchanged) passing false.  Doing something so that we only have to modify the places currently passing true, seems better.  Some choices are:
1. Have two signatures, restaurants() and restaurantsWithFormulaAudit()
2. Have a single restaurants() method with no params, and set validateForumlas state in an alternate constructor,  by adding an enum and having a second ctor that takes the enum, and have the default ctor set the state to the non-audit value.

I like the second one a little bit better.